src/formatting.ts

```diff
@@ -34,7 +45,10 @@ export const convertObjectToCamelCase = (
 
 export const toFormData = (object: SimpleObject): FormData => {
   return Object.entries(object).reduce((formData, [key, value]) => {
-    if (value !== undefined && value !== null) {
+    if (value instanceof Object && "contents" in value) {
+      const { contents, ...options } = value as ContentsAndOptions;
+      formData.append(snakeCase(key), contents, options);
+    } else if (value !== undefined && value !== null) {
```

Comment:
What happens here when `value` is any other `falsy` value?

Reply:
form-data converts it to a string and then the backend converts it back to whatever type it's expecting for that field.